docs/csharp/misc/cs0841.md

```diff
@@ -1,35 +1,49 @@
 ---
 title: "Compiler Error CS0841"
 ms.date: 07/20/2015
-f1_keywords: 
+f1_keywords:
   - "CS0841"
-helpviewer_keywords: 
+helpviewer_keywords:
   - "CS0841"
 ms.assetid: eb67c244-a930-4291-ae2a-5832e8916ed7
 ---
 # Compiler Error CS0841
-Cannot use variable 'name' before it is declared.  
-  
- A variable must be declared before it is used.  
-  
-## To correct this error  
-  
-1. Move the variable declaration before the line where the error occurs.  
-  
-## Example  
- The following example generates CS0841:  
-  
-```csharp  
-// cs0841.cs  
+
+Cannot use local variable 'name' before it is declared.
```

Comment:
I added the word "local" here because this is the exact message generated by Roslyn.
https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/CSharpResources.resx#L2229